### DISCUSSION TITLE
Allow bypassing required pull requests

### DISCUSSION THREAD

**Discussion by electriquo** [community/community#7958] [Pull Requests]
Hi,
Looking at the [roadmap](https://github.com/github/roadmap/projects/1), there is no indication if:
1. Is there any plan to support [Allow bypassing required pull requests](https://github.blog/changelog/2021-11-19-allow-bypassing-required-pull-requests/) in [REST API](https://docs.github.com/en/rest/reference/repos#update-branch-protection)?
2. If so, then when?

**Comment by wagnst** [comment]
Also we need this in the Terraform Provider!!!
